tests/appsec/iast/conftest.py

```diff
@@ -162,11 +165,27 @@ def check_native_code_exception_in_each_python_aspect_test(request, caplog):
 @pytest.fixture(scope="session")
 def configuration_endpoint():
     current_dir = os.path.dirname(__file__)
-    cmd = [
-        "python",
-        os.path.join(current_dir, "fixtures", "integration", "http_config_server.py"),
-        CONFIG_SERVER_PORT,
-    ]
-    process = subprocess.Popen(cmd, cwd=current_dir)
+    status = None
+    retries = 0
+    while status != 200 and retries < 5:
+        cmd = [
+            "python",
+            os.path.join(current_dir, "fixtures", "integration", "http_config_server.py"),
+            CONFIG_SERVER_PORT,
+        ]
+        process = subprocess.Popen(cmd, cwd=current_dir)
+        time.sleep(0.1)
```

Comment:
is half a second enough time for subprocess to come up?

Reply:
If I understand correctly, [subprocess.Popen returns when the child process has been created](https://github.com/python/cpython/blob/main/Lib/subprocess.py#L1035). We're adding 0.1 seconds to wait until it's really ready to respond to requests, maybe it's not enough, but we're currently not waiting at all and it works most of the time.